ci/sync-version.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+#
+# GRAKN.AI - THE KNOWLEDGE GRAPH
+# Copyright (C) 2018 Grakn Labs Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import sys
+import logging
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(
+    format='[%(asctime)s.%(msecs)03d]: %(message)s',
+    datefmt='%Y-%m-%d %H:%M:%S',
+)
+logger.level = logging.DEBUG
+
+if len(sys.argv) != 2:
+    raise Exception('Should pass version file as first argument')
+
+version = None
+version_fn = sys.argv[1]
+
+with open(version_fn) as version_file:
+    version = version_file.read().strip().split('.')
+
+if len(version) != 3:
+    raise Exception('Version file should contain \'major.minor.patch\'')
```

Comment:
How about `Version file should be in the following format: 'x.y.z'`? I think major, minor, patch isn't the correct term for us, semantically

Reply:
@lolski fixed in https://github.com/graknlabs/build-tools/pull/48/commits/e62219b60e5a0c520985cf60cc85d649487d3439